python/ray/workflow/api.py

```diff
@@ -41,11 +41,17 @@
 
 
 @PublicAPI(stability="beta")
-def init() -> None:
+def init(*, max_running_workflows: int = -1, max_pending_workflows: int = -1) -> None:
```

Comment:
should the default one be None?

I'm thinking about the use case where the cluster is up and the workflow actor is up. Then the user wants to submit the jobs. THen does this mean the user has to always specify the right number?